jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/Etcd.java

```diff
@@ -46,6 +46,7 @@ public static class Builder {
         private boolean ssl = false;
         private List<String> additionalArgs;
         private Network network;
+        private boolean bindVolumn = true;
```

Comment:
- the default value is usually set in the constructor
- maybe use a better name that describe the intention i.e. bindDataDirectory